difficulties.lua

```diff
@@ -24,6 +43,19 @@ function HandleDifficultyCommand ( Split, Player )
 	if (Split[2] == "peaceful") or (Split[2] == "0") or (Split[2] == "p") then
 		SetWorldDifficulty(Player:GetWorld(), 0)
 		SendMessage( Player, "World difficulty set to peaceful" )
+		
+		--Remove mobs which are not allowed in peaceful
+		Player:GetWorld():ForEachEntity(
+			function (a_Entity)
+				if not(a_Entity:IsMob()) then
+					return
+				end
+				local Monster = tolua.cast(a_Entity, "cMonster")
+				if (Monster:GetMobFamily() == cMonster.mfHostile) then
+					Monster:Destroy()
+				end
+			end
+			)

```

Comment:
I know I'm being a nitpicker, but now this parenthesis has an extra indent level. It should be on the same level as the `ForEachEntity` call which it closes.

Of course I could fix that, too, but not in this PR (not allowed by GitHub), I'd need to create an extra commit  just for the fix after merging this PR, which would make the history messy.